data_models/bioinformatics_mill_models/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.utils import timezone
+
+
+class TimeTrackedModel(models.Model):
+    created_at = models.DateTimeField(editable=False)
+    updated_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
```

Comment:
Ok, as we discussed in person, I think there are both benefits and drawbacks from both. Im ok with sticking with this option for now, and we can discuss some more about it after the PR has been merged.